qutip/tests/core/test_expect.py

```diff
@@ -154,7 +154,7 @@ def test_equivalent_to_matrix_element(hermitian):
     op = qutip.rand_herm(dimension, 0.2)
     if not hermitian:
         op = op + 1j*qutip.rand_herm(dimension, 0.1)
-    expected = (state.dag() * op * state).data[0, 0]
```

Comment:
Not sure if it was what I had in mind anymore, but I expect your removed the scipy 1.13.0 patch where for that release only csr subscription returned 0d numpy array instead of number.

Reply:
Yeah, it changes in 45992d5.